File: akka-scala/src/main/scala/io/swagger/client/model/CloudTable.scala
package io.swagger.client.model

import io.swagger.client.core.ApiModel
import org.joda.time.DateTime


case class CloudTable (
  name: Option[String],
  appId: Option[String],
  `type`: Option[CloudTableEnums.`Type`],
  columns: Option[Seq[Column]])
   extends ApiModel

object CloudTableEnums {

  type `Type` = `Type`.Value
  
  object `Type` extends Enumeration {
    val User = Value("user")
    val Role = Value("role")
    val Custom = Value("custom")
  }

  
}

File: typescript-angular/API/Client/CloudTable.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface CloudTable {

        "name"?: string;

        "appId"?: string;

        "type"?: CloudTable.TypeEnum;

        "columns"?: Array<Column>;
    }

    export namespace CloudTable {

        export enum TypeEnum { 
            user = <any> 'user',
            role = <any> 'role',
            custom = <any> 'custom'
        }
    }
}
